### DISCUSSION TITLE
Welcome to SimpleTuner Discussions!

### DISCUSSION THREAD

**Discussion by bghira** [bghira/SimpleTuner#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davizca** [comment]
Gonna test this today with my 24 GB VRAM 3090 RTX to see if I can finetune at batch size 8 or lower than 8 a set of images I have over here. Will keep you informed. I have my Ubuntu distro ready

**Reply by bghira** [reply]
for a 3090 with 24GB VRAM you'll likely want a batch size of 1, and to use DeepSpeed ZeRO stage 1 or stage 2 - the lower of which is better. if ZeRO stage 1 works for you, don't use stage 2. But, use stage 2 if stage 1 ends up OOM'ing.

a batch size of 1 ensures that the model parameters and optimiser states (where possible) fit into VRAM more easily.

if you use a batch size of 8, then the accelerator has to spend more time offloading components.

**Reply by davizca** [reply]
I think I did everything right I only miss one thing. I have captions, where do I have to edit and what to put to use captions as txt.

And also I had this [message of error](https://gyazo.com/6c6c546629e1b7ba5e3f623c6ff18b1c)
